payroll/models/hr_payslip.py

```diff
@@ -623,9 +623,13 @@ def get_lines_dict(self):
                 for rule in payslip._get_salary_rules():
                     localdict = rule._reset_localdict_values(localdict)
                     # check if the rule can be applied
-                    if rule._satisfy_condition(localdict) and rule.id not in blacklist:
+                    rule_sudo = rule.sudo()
```

Comment:
Why is this running with admin privileges? If anything, doesn't this make it less secure? Before it was running as Payroll Manager now it's running as Administrator.

Reply:
I didn't saw this change. Why is this sudo necessary now?